src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -939,21 +939,17 @@ protected override void EndProcessing()
 
                 // Now open the output file...
                 PathUtils.MasterStreamOpen(
-                    this,
-                    filePath,
-#if UNIX
-                    new UTF8Encoding(false), // UTF-8, no BOM
-#else
-                    EncodingConversion.Unicode, // UTF-16 with BOM
-#endif
-                    /* defaultEncoding */ false,
-                    /* Append */ false,
-                    /* Force */ false,
-                    /* NoClobber */ false,
-                    out fileStream,
-                    out streamWriter,
-                    out readOnlyFileInfo,
-                    false
+                    cmdlet : this,
+                    filePath : filePath,
+                    resolvedEncoding : new UTF8Encoding(false), // UTF-8, no BOM
```

Comment:
should be `new UTF8Encoding(encoderShouldEmitUTF8Identifier: false)`

Reply:
will update, missed that one